### ISSUE TITLE
feature: supports js

### ISSUE THREAD

**Issue by Allen-1998** [Allen-1998/pinia-auto-refs#3]
In some js projects, there are also developers who want to use this plugin to make the project better. But I haven't thought of anything more elegant than providing two different templates and a single configuration item, so open the issue up for discussion.

**Comment by hermit99** [comment]
I believe the author's [later solution](https://github.com/vuejs/pinia/issues/718#issuecomment-1103658199) is much better than the former one, which this package implemented on. It's a thin wrapper without the fuss of generating helper files, but suffice the need.

**Comment by Allen-1998** [comment]
Yes, I agree with you, just try different ways to work [like this view](https://github.com/vuejs/pinia/issues/718#issuecomment-941979177). The best of course is to hope that pinia can integrate a similar approach to the author into core
